server/models/Profile.js

```diff
@@ -38,7 +38,24 @@ phone: {
     message: '{VALUE} is not a valid 10 digit number!'
   }
 },
-availability : { type: Schema.Types.ObjectId, ref: 'Availability' },
+availability : {
+  availabilityDays: {
+    type: Array
+  },
+  availabilityDates: [{
+    day: {
+      type: Date,
```

Comment:
We have a date for 'day' and have a date range within 'periods' as well, I'm not sure how this design will be implemented to fit our project use case but I'll approve this PR so that we can get the Profile Model in :)